src/components/Image/Image.js

```diff
@@ -36,15 +40,17 @@ const getValidSource = (source: FastImageSource | number) => {
 // This can be refactored to FC as soon as we update Styled Componenents.
 
 // eslint-disable-next-line react/prefer-stateless-function
-class Image extends React.Component<ImageProps> {
+class Image extends React.Component<ImageProps, Props> {
```

Comment:
It looks like you passed `Props` type to generic param for `State` type:
```
React$Component<Props, State = void>
```

Instead of that, please see the comment next to `Props` definition